coffea/jetmet_tools/CorrectedJetsFactory.py

```diff
@@ -140,7 +156,11 @@ def __init__(self, name_map, jec_stack):
 
         self.real_sig = [v for k, v in name_map.items()]
         self.name_map = name_map
-        self.jec_stack = jec_stack
+        self.jec_stack = jec_stack  ## to be removed after the porting is completed
+        self.jec_names = jec_names
+        self.jec_year = jec_year
+        ## the last element of jec_names has to be a boolean, specifying if the use wants to save the different correction levels
+        self.separated = self.jec_names.pop()
```

Comment:
This could be a useful debug feature, but from the JME point of view, providing just the name of the JEC tag for the required step should suffice. I don't foresee a need for analyses to store the result of each step, but I'm open to hearing about any exceptions.